kahuna/public/js/search/results.js

```diff
@@ -154,6 +154,9 @@ results.controller('SearchResultsCtrl', [
             }
 
             return images;
+        }).catch(error => {
+            ctrl.loadingError = error;
```

Comment:
Yes, I think we need to think of a way to bring errors back into their location of origin (the button triggered, the area not rendered) instead of always having global errors.